app/controllers/api/v0/instances_controller.rb

```diff
@@ -25,6 +25,24 @@ def visibility
 
         Settings::UserExperience.public ? "public" : "private"
       end
+
+      def release_version
+        File.read(Rails.root.join(".release-version"))
+
+      # Accommodate the .release-version file not existing in the case where
+      # this deployment is deployed from a checkout/snapshot of the code.
+      rescue StandardError
+        # Get the latest modified file in the app. We don't use git in case it's
```

Comment:
Could we ensure we always have a release version file, e.g. by updating it via a GH action when merging to `main`? It's probably not worth the effort but it does seem a bit "cleaner" in the sense that if we do it right we'd also have correct suffixes like `.1`.

Reply:
This is a really good idea! I'm not in love with this implementation as-is, but it gets us there, so I'll probably merge it as-is to at least get this out the door for @forem/ecosystem, but I definitely want to look into your idea!